ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.15__amd64-nightly.yaml

```
@@ -2032,6 +2032,34 @@ tests:
     test:
     - chain: openshift-e2e-test-hypershift-qe-mgmt
     workflow: cucushift-installer-rehearse-baremetalds-ipi-ovn-ipv4-kubevirt-hypershift-mce
+- as: baremetalds-ipi-ovn-kubevirt-hypershift-mce-n1minor-mgmt-f28
+  cron: 10 20 12 * *
+  steps:
+    cluster_profile: equinix-ocp-metal-qe
+    dependencies:
+      NODEPOOL_RELEASE_IMAGE_LATEST: release:n1minor
+    env:
+      LVM_OPERATOR_SUB_CHANNEL: stable-4.15
+      MCE_VERSION: "2.5"
+      ODF_OPERATOR_SUB_CHANNEL: stable-4.15
+    test:
+    - chain: openshift-e2e-test-hypershift-qe-mgmt
+    workflow: cucushift-installer-rehearse-baremetalds-ipi-ovn-ipv4-kubevirt-hypershift-mce
+- as: baremetalds-ipi-ovn-kubevirt-hypershift-mce-n1minor-guest-f14
+  cron: 23 14 22 * *
```

should be ?
```suggestion
  cron: 35 15 12,26 * *
```
It should be consistent with the frequency `f14` or the frequency should set to `f28`.
```console
$ ./ci-operator/config/openshift/openshift-tests-private/tools/generate-cron-entry.sh baremetalds-ipi-ovn-kubevirt-hypershift-mce-n1minor-guest-f14 openshift-openshift-tests-private-release-4.15__amd64-nightly.yaml
cron: 35 15 12,26 * *
```